--- faqs.html.hbs	2022-04-05 11:53:14.000000000 -0400
+++ faqs.updated.html.hbs	2022-04-05 11:54:18.000000000 -0400
@@ -15,7 +15,7 @@
     {{!-- {{> templates/vertical-standard/script/sortoptions }} --}}
     {{!-- {{> templates/vertical-standard/script/facets }} --}}
     {{!-- {{> templates/vertical-standard/script/filterbox }} --}}
-    {{!-- {{> templates/vertical-standard/script/qasubmission }} --}}
+    {{> templates/vertical-standard/script/qasubmission }}
   {{/script/core }}
   <div class="Answers AnswersVerticalStandard CollapsibleFilters">
     <div class="Answers-navWrapper">
@@ -50,7 +50,7 @@
         <div class="Answers-results js-answersResults">
           {{> templates/vertical-standard/markup/verticalresults }}
           {{> templates/vertical-standard/markup/pagination }}
-          {{!-- {{> templates/vertical-standard/markup/qasubmission }} --}}
+          {{> templates/vertical-standard/markup/qasubmission }}
         </div>
       </div>
     </div>
